selfdrive/ui/qt/offroad/wifiManager.cc

```diff
@@ -212,8 +212,6 @@ void WifiManager::connect(const Network &n, const QString &password) {
 
 void WifiManager::connect(const Network &n, const QString &username, const QString &password) {
   connecting_to_network = n.ssid;
-  // disconnect();
-  clear_connections(n.ssid); //Clear all connections that may already exist to the network we are connecting
```

Comment:
why remove this?

Reply:
connect is now only reached if it's a new connection, `clear_connections` actually deleted the connection, not disconnected from it

Reply:
We can uncomment the disconnect line though, that will only deactivate the connection if we're currently connected to a network